modules/ROOT/pages/api-groups-deleting-groups.adoc

```diff
@@ -0,0 +1,19 @@
+= Delete API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You might want to delete a API Group if it is no longer required based on your business needs.
```

Comment:
The user deletes group instances. If the user wants to delete the group he needs to delete every group instance fo the group one by one

Reply:
Done! Thanks!